app/controllers/devise/devise_authy_controller.rb

```diff
@@ -14,6 +14,8 @@ class Devise::DeviseAuthyController < DeviseController
 
   def GET_verify_authy
     @authy_id = @resource.authy_id
+    approval_request = send_one_touch_request['approval_request']
```

Comment:
Can we make this optional? Many customers already are using this and decided not to have one_touch enabled, but only relying on SMSs or Calls.
A flag should be enough.

Reply:
Done